examples/src/main/scala/com/github/fit51/reactiveconfig/examples/EtcdConfigApplication.scala

```diff
@@ -36,26 +38,35 @@ object EtcdConfigApplication extends App {
   implicit val scheduler = Scheduler.global
 
   def init[F[_]: ContextShift: Async: Concurrent: Timer: TaskLike](
+      etcdClient: EtcdClient[F] with Watch[F],
       goods: Map[ProductId, Count]
   )(implicit scheduler: Scheduler): F[CommandLineShopService[F]] = {
     for {
       _      <- FillConfig.fill
-      config <- ReactiveConfigEtcd[F, Json](ChannelManager.noAuth("http://127.0.0.1:2379"))
+      config <- ReactiveConfigEtcd[F, Json](etcdClient)
 
-      storeConfig = config.reloadable[StoreConfig]("store.store")
+      storeConfig <- config.reloadable[StoreConfig]("store.store")
       implicit0(storeService: StoreService[F]) <- StoreModule.StoreService[F](goods, storeConfig)
 
-      advertsList = config.reloadable[List[ProductId]]("store.adverts")
-      advertsConfig = storeConfig.combine(advertsList) { (store, list) =>
+      advertsList <- config.reloadable[List[ProductId]]("store.adverts")
+      advertsConfig <- storeConfig.combine(advertsList) { (store, list) =>
         AdvertsConfig(store.priceList, list)
       }
       implicit0(advertsService: AdvertsService[F]) = new AdvertsService[F](advertsConfig)
     } yield new CommandLineShopService[F]()
   }
 
-  val f = init[Task](FillConfig.store).flatMap(_.flow).runToFuture
-
-  Await.result(f, Duration.Inf)
+  val chManager = ChannelManager.noAuth("http://127.0.0.1:2379")
+  val future =
+    (for {
+      client <- Task.pure(new EtcdClient[Task](chManager) with Watch[Task] {
+        override val taskLift = TaskLift[Task]
+      })
```

Comment:
EtcdClient with Watch initialization looks cumbersome.
I suggest to add another method to EtcdClient companion object, that would require TaskLift and return EtcdClient with Watch.

Reply:
Previously ReactiveConfigEtcd played a builder role. Now there is no way to avoid manual cumbersome initialization.
Suppose we should retrieve ReactiveConfigEtcd apply method, that would create EtcdClient with Watch under the hood.

Reply:
Done.